awx/main/tests/functional/dab_rbac/test_dab_rbac_api.py

```diff
@@ -88,3 +89,32 @@ def test_assign_custom_add_role(admin_user, rando, organization, post, setup_man
     inv_id = r.data['id']
     inventory = Inventory.objects.get(id=inv_id)
     assert rando.has_obj_perm(inventory, 'change')
+
+
+@pytest.mark.django_db
+def test_jt_creation_permissions(setup_managed_roles, inventory, project, rando):
+    """This tests that if you assign someone required permissions in the new API
+    using the managed roles, then that works to give permissions to create a job template AAP-25603"""
+    inv_rd = RoleDefinition.objects.get(name='Inventory Admin')
+    proj_rd = RoleDefinition.objects.get(name='Project Admin')
+    # establish prior state
+    access = JobTemplateAccess(rando)
+    assert not access.can_add({'inventory': inventory.pk, 'project': project.pk, 'name': 'foo-jt'})
+
+    inv_rd.give_permission(rando, inventory)
+    proj_rd.give_permission(rando, project)
+
+    assert access.can_add({'inventory': inventory.pk, 'project': project.pk, 'name': 'foo-jt'})
+
+
+@pytest.mark.django_db
+def test_workflow_creation_permissions(setup_managed_roles, organization, workflow_job_template, rando):
+    """Similar to JT, assigning new roles gives creator permissions AAP-25635"""
```

Comment:
Calling out to @PabloHiro how I'm marking these. I don't believe we have any formal, structural, way to do these links and I'm fine with that.

Reply:
I am not entirely sure if we should even reference internal JIRA issues in the upstream. I have been grepping the repo and the only reference I have found to an issue is here: https://github.com/ansible/awx/blame/devel/tools/ansible/roles/dockerfile/templates/Dockerfile.j2#L39

@TheRealHaoLiu opinions?

Reply:
I would reference the issues directly in the PR description maybe? That way nothing downstream creeps up into upstream code, but we can still trace stuff.

Reply:
I thought it would be nice to map the _specific test case_ to the issue. I'll list them in the PR body, sure. But with 4 related issues here, I think that might be a bit much to follow along with.

Reply:
Markers have been removed.